boards/PDM/Test/Src/Test_StateMachine.cpp

```diff
@@ -189,3 +366,30 @@ TEST_F(PdmStateMachineTest, check_air_closed_state_is_broadcasted_over_can)
         CANMSGS_PDM_STATE_MACHINE_STATE_AIR_CLOSED_CHOICE,
         App_CanTx_GetPeriodicSignal_STATE(can_tx_interface));
 }
+
+TEST_F(PdmStateMachineTest, check_vbat_voltage_can_signals_in_all_states)
```

Comment:
So the idea here is  to have 
- 1 test for each voltage (because the bheaviour is the same in every state)
- 2 tests for each current (because the behaviour in INIT state is different than other states)